cashctrl_ledger/ledger.py

```diff
@@ -290,15 +395,16 @@ def mirror_account_chart(self, target: pd.DataFrame, delete: bool = False):
             delete (bool, optional): If True, deletes accounts on the remote that are not
                                      present in the target DataFrame.
         """
+        if target is not None:
+            target = target.copy()
```

Comment:
This is not because of `standardize_account_chart()` method.

This caused by:
```python
        def account_groups(df: pd.DataFrame) -> Dict[str, str]:
            if df is None or df.empty:
                return {}

            df["nodes"] = [
                pd.DataFrame({"items": get_nodes_list(path)}) for path in df["group"]
            ]
```

I propose to add a copy here before creating new column: 
```python
        def account_groups(df: pd.DataFrame) -> Dict[str, str]:
            if df is None or df.empty:
                return {}

            df = df.copy()
            df["nodes"] = [
                pd.DataFrame({"items": get_nodes_list(path)}) for path in df["group"]
            ]
```

Reply:
Agreed. Thanks for investigating!